test/extended/networking.sh

```diff
@@ -7,9 +7,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [[ "${OPENSHIFT_QUIET_OUTPUT:-false}" != "true" ]]; then
+CONFIG_ROOT="${OPENSHIFT_CONFIG_ROOT:-}"
+
+# CI mode (deploying clusters for each plugin) should be noisy by
+# default to aid in debugging.
+if [[ "${CONFIG_ROOT}" = "" ]]; then
+  DEFAULT_QUIET=false
+else
+  DEFAULT_QUIET=true
+fi
+
+if [[ "${OPENSHIFT_QUIET_OUTPUT:-${DEFAULT_QUIET}}" != "true" ]]; then
```

Comment:
I think there's a logical error in the proposed logic.  Why should the the following test indicate that verbose output was called for? 

    -n ${OPENSHIFT_QUIET_OUTPUT:-}

Here's the truth table of the proposed logic:
 
|   config root  | quiet           | verbose  |
| ------------- |:-------------:| -----:|
| unset      |  unset | true  |
| unset      | set      |  true  |
| set      | unset      | false |
| set      | set      |   true |

Even were that logic fixed, the truth table for a binary quiet would be:

|   config root      | quiet           | verbose  |
| ------------- |:-------------:| -----:|
| unset      |  unset | true  |
| unset      | set      |  false  |
| set      | unset      | false |
| set      | set      |   false |

Assuming I want to be able to enable logging when config root is set, I think quiet would need to be a ternary value:

|   config root      | quiet           | verbose  |
| ------------- |:-------------:| -----:|
| unset      |  unset | true  |
| unset      | true      |   false |
| unset   | false      |   true |
| set     | unset      |   false |
| set     | true      |   false |
| set      | false     |   true|

Clearly this is too complicated, though.  I'm going to simplify so that verbose output is controlled by a single binary flag.